README.md

```diff
@@ -1 +1,37 @@
-# node-patterns-concurrency-lab1
+# Async Lab
+
+## Objectives
+
+1. Implement your own async loop
+1. You cannot use async or some other library
+1. You need to make 10 requests to another service and wait until all of them responded before rendering
+1. Utilize Storify API to fetch images
+
+## Introduction
+
+You are hired to create a website that sells stock images for designers. In order for this website to have inventory, you need to implement a scraper which fetches images from Instagram. You'll need to fetch images from some IG accounts (links provided) and save them on the disk.
+
+In this lab, you'll implement asynchronous loop without any extra libraries to manage asynchronous code. You goal is to create a fast implementation so the requests must be fired as soon as possible, not in sequential order.
```

Comment:
The requests are still fired in sequential order, but they can return out of order.